src/SB/Game/zEntPlayer.cpp

```diff
@@ -3639,7 +3693,7 @@ void zEntPlayerPreReset()
 F32 ComputeFudge(F32 a, F32 b)
 {
     F32 min = MIN(a, b);
-    a = (min - 0.075f) / -0.175f;
+    a = (min - -0.175f) / 0.074999996f; // Will not match with 0.075f.
```

Comment:
This could be `a = (min - -0.175f) * 13.33333333f`, try to experiment with that and how to round the .33333... and see if it even becomes a division instead of multiplication.

Reply:
No luck, but I realised `(3.0f / 40.0f)` instead of `0.074999996f` also works. Not really sure if that's really "better" clarity-wise, though.